File: frontend/src/app/dto/attendance/attendance.update.req.dto.ts
export class AttendanceUpdateReqDto {
    private id! : number
    private approval! : boolean
    private ver! : number

    public get _approval(): boolean {
        return this.approval;
    }

    public set _approval(_approval: boolean) {
        this.approval = _approval;
    }

    public get _id(): number {
        return this.id;
    }

    public set _id(_id: number) {
        this.id = _id;
    }

    public get _ver(): number {
        return this.ver;
    }

    public set _ver(_ver: number) {
        this.ver = _ver;
    }
    
}

File: src/main/java/com/lawencon/elearn/dto/attendance/AttendanceUpdateReqDto.java
package com.lawencon.elearn.dto.attendance;

public class AttendanceUpdateReqDto {
	private Long id;
	private Boolean approval;
	private Integer ver;
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public Boolean getApproval() {
		return approval;
	}
	public void setApproval(Boolean approval) {
		this.approval = approval;
	}
	public Integer getVer() {
		return ver;
	}
	public void setVer(Integer ver) {
		this.ver = ver;
	}
}
